projects/cypress/CY8CKIT_064S0S2_4343W/mtb/aws_demos/afr.mk

```diff
@@ -331,6 +333,20 @@ INCLUDES+=\
 	$(CY_AFR_ROOT)/libraries/coreMQTT/source/include\
 	$(CY_AFR_ROOT)/libraries/coreMQTT/source/portable\
 
+# include device-shadow-for-aws-iot-embedded-sdk
+SOURCES+=\
```

Comment:
I think it would be better to add these at the end of the SOURCES list after line 308 and the end of the INCLUDES list after line 334